web/turkey/survey/static/survey/js/auditors/scrolled_pixels_specifics.js

```diff
@@ -0,0 +1,48 @@
+var AuditorScrolledPixelsSpecific = {
+    scrolled_pixels_specific: [],
+    previous_position_h: 0,
+    previous_position_v: 0,
+    setup: function() {
+        this.previous_position_h = $(window).scrollLeft();
+        this.previous_position_v = $(window).scrollTop();
+    }
+    log_scroll_specific: function () {
+        // horizontal
+        var current_position_h = $(window).scrollLeft();
+        var raw_amount_h = current_position_h - previous_position_h;
+        var horizontal = { 'position' : current_position_h, 'change' : raw_amount_h };
+        previous_position_h = current_position_h;
+
+        // vertical
+        var current_position_v = $(window).scrollTop();
+        var raw_amount_v = current_position_v - previous_position_v;
+        var vertical = { 'position' : current_position_v, 'change' : raw_amount_v };
+        previous_position_v = current_position_v;
+
+        this.scrolled_pixels_specific.push(
+            {
+                'horizontal'    : horizontal,
```

Comment:
I like timestamps. Would this apply to auditors for clicking, keypressing, etc as well or did you mean just for scrolling auditors? I'm thinking it would probably make more sense if probably everything had a time stamp

(Off topic, but we could present a chronological list of all the user's actions chronologically; maybe something we could look into at the very end? or not lol)